cmake/SSGCommon.cmake

```diff
@@ -769,7 +765,13 @@ macro(ssg_build_product PRODUCT)
     ssg_build_xccdf_unlinked(${PRODUCT})
     ssg_build_ocil_unlinked(${PRODUCT})
     ssg_build_remediations(${PRODUCT})
-    if ("${PRODUCT_ANSIBLE_REMEDIATION_ENABLED}")
+
+    add_custom_target(
+        generate-${PRODUCT}-ansible-playbooks
+        DEPENDS "${ANSIBLE_PLAYBOOKS_DIR}"
```

Comment:
I think the `DEPENDS` needs to stay within `ssg_build_ansible_playbooks`, as at this point it was not defined yet.